lib/request.ts

```diff
@@ -12,12 +12,25 @@ const getInterval = (time: [number, number]): number => {
   return Math.round(diff[0] * 1000 + diff[1] / 1000000);
 };
 
+/**
+ * The NodeJS typescript definitions has OutgoingHttpHeaders
```

Comment:
Short term this is a good fix, but ideally we should not support extra definitions. Could you please add an issue there and link it in the comment?

Reply:
updated description

Reply:
Could you please also add the issue link here?